model/src/pyrenew/latent/hospitaladmissions.py

```diff
@@ -16,54 +15,153 @@
 )
 """Output from HospitalAdmissions.sample()"""
 
+InfectHospRateSample = namedtuple(
+    "InfectHospRateSample",
+    ["IHR"],
+    defaults=[None],
+)
+
+
+class InfectHospRate(RandomVariable):
+    """Infection to Hospitalization Rate"""
+
+    def __init__(
+        self,
+        varname: str = "IHR",
+        dist: dist.Distribution = dist.LogNormal(jnp.log(0.05), 0.05),
```

Comment:
Shouldn't preserve the default now that this is modular; default was reasonable in the `basic.py` example, but this should have no default.

Reply:
Addressed in [d0851f9](https://github.com/CDCgov/multisignal-epi-inference/pull/39/commits/d0851f9f3c3289950f4fca8c855a67fe36ad8ed9)